components/navigation/Filter.js

```diff
@@ -0,0 +1,38 @@
+import {useState} from 'react';
+import PropTypes from 'prop-types';
+
+export default function Filter({data, onFilter}) {
+    const [state, setState] = useState('Add filter');
```

Comment:
```suggestion
    const [state, setState] = useState('Filter');
```

I think just Filter is fine

Reply:
@derberg I've updated the PR, please kindly review.